index.js

```diff
@@ -67,6 +67,15 @@ let cachedRecords;
 let credentials;
 
 /**
+ * Maps between passed in user name and internal user name.
```

Comment:
I cleaned up the documentation compared to the previous: https://github.com/sugarcrm/thorn/pull/122/files#diff-168726dbe96b3ce427e7fedce31bb0bcR66